prime-router/docs/openapi.yml

```diff
@@ -9,6 +9,399 @@ info:
+        - in: query
+          name: dateEnd
+          schema:
+            oneOf:
+              - type: integer
+              - type: string
+          required: false
+          description: (`YYYY-MM-DD` || `NNN` - default `0`) The end date or the number of days back from today to collect results from.
+        - in: query
+          name: year
+          schema:
+            type: integer
+          required: false
+          description: (`YYYY` - default `ALL`) Return results from specific Year.
+        - in: query
+          name: month
+          schema:
+            type: integer
+          required: false
+          description: (`MM` - default `ALL`) Return results from specific Month. Requires `year` being set.
+        - in: query
+          name: organizations
+          schema:
+            oneOf:
+              - type: array
+              - type: integer
+          required: false
+          description: (`{INT, INT, INT}` || `INT` - default `ALL`) Return results from specific Organization or array of Organizations. Admin or member of multiple Organizations required.
+        - in: query
+          name: recipients
+          schema:
+            oneOf:
+              - type: array
+              - type: integer
+          required: false
+          description: (`{INT, INT, INT}` || `INT` - default `ALL`) Return results from specific Recipient or array of Recipients.
+        - in: query
+          name: senders
+          schema:
+            oneOf:
+              - type: array
+              - type: integer
+          required: false
+          description: (`{INT, INT, INT}`  || `INT` - default `ALL`) Return results from specific Sender or array of Senders.
+        - in: query
+          name: warnings
+          schema:
+            type: string
+          required: false
+          description: (`ALL` || `WITH` || `WITHOUT` - default `ALL`) If `WITH` only return results with Warning(s). If `WITHOUT` only results without Warning(s).
+        - in: query
+          name: errors
+          schema:
+            type: string
+          required: false
+          description: (`ALL` || `WITH` || `WITHOUT` - default `ALL`) If `WITH` only return results with Error(s). If `WITHOUT` only results without Error(s).
+        - in: query
+          name: receipts
+          schema:
+            type: string
+          required: false
+          description: (`ALL` || `WITH` || `WITHOUT` - default `ALL`) If `WITH` only return results with Receipt(s). If `WITHOUT` only results without Receipt(s).
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Report'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+
+
+  /submissions/multiorg:
+    get:
```

Comment:
I'm not convinced that `multiorg` is needed if it is the same as the `submissions` org. From a resource name perspective, it seems redundant.